ddev/src/ddev/cli/release/branch/create.py

```diff
@@ -0,0 +1,53 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import annotations
+
+import re
+from typing import TYPE_CHECKING
+
+import click
+
+if TYPE_CHECKING:
+    from ddev.cli.application import Application
+
+BRANCH_NAME_PATTERN = re.compile(r'^\d+\.\d+\.x$')
+GITHUB_LABEL_COLOR = '5319e7'
+
+
+@click.command
+@click.pass_obj
+@click.argument('branch_name')
+def create(app: Application, branch_name):
+    """
+    Create the release branch.
+
+    This command will also create the `backport/` label in GitHub for this release branch.
```

Comment:
```suggestion
    This command will also create the `backport/AGENT_VERSION` label in GitHub for this Agent release.
```

Reply:
`7.52.x` is not the Agent version, it's the branch's name

Reply:
I don't get it, it's a branch. Why use another word?

Reply:
If you tell me `ddev release branch create <name>` I'm surprised that the command `ddev release branch create foo` fails ;)

If you tell me `ddev release branch create <agent_release_group>`, I'm much less surprised by `foo` failing. 

It's an extreme example, but my point is that "branch name" doesn't explain the restrictions we apply on this branch name.

Big picture I guess as long as we give some hint in the help for the correct branch format it would be fine. Doesn't have to be argument name, can also be the exact pattern we match against.

Reply:
It will fail with [this error message](https://github.com/DataDog/integrations-core/pull/16905/files#diff-e86578b2aab0cba83981087aa3abde0d3ca18e0925ffee99f78d38ad1d17b92dR30), but I can add it to the docs as well

Reply:
Let's do that. Imagine somebody new to the team who's just told to run the command by the confluence docs. Keeping the format in the docs here will at least get them started.

Reply:
Added!